trees/utils/StrongTreeFairOCT.py

```diff
@@ -0,0 +1,284 @@
+'''
+This module formulate the FlowOCT problem in gurobipy.
```

Comment:
Is this formulation only used for FairTrees? If so, I think we should rename this file to make it clear this is not used for StrongTrees